### PR TITLE
add web support

### PR DESCRIPTION
**Pull request by sreehax**

### CODE REVIEW

**Review by anirudhb — APPROVED** [review]
LGTM

File: lib/constants.dart [hunk 1]
```diff
@@ -5,7 +5,7 @@ class EnboxConstants extends InheritedWidget {
 
   final String displayName = "enbox";
   final Color color = Colors.blue;
-  final String demoName = "Enbox Demo";
+  final String demoName = "Enbox Demo 2.0";
```
Comment (Treeways) [inline lib/constants.dart:8]: Is this appropriate?

Comment (sreehax) [reply lib/constants.dart:8]: this is just demo data anyways, and i just changed this to indicate a successful automated build at mail.enbox.app, so we know it updated

Comment (anirudhb) [reply lib/constants.dart:8]: Sure, since it's on web.

Comment (Treeways) [reply lib/constants.dart:8]: Got it.

**Pull request merged**